mappings/remediation_advice.json

```diff
@@ -307,7 +307,7 @@
           "children": [
             {
               "id": "brute_force",
-              "remediation_advice": "Login: When an authentication request fails, ensure that the error or message and response is identical, regardless of whether it was the username or the password that caused the authentication to fail.  The HTTP response needs to be the same as well as there not being a single byte or bit difference.\n\nReset Password: There can not be a discrepancy when the form is submitted to reset a password.  The same message should be sent regardless of whether the username or email exists as an account or not.\n\nRegistration: As a best practice to prevent usernames from being enumerated when creating an account, the username should an email address.  When an attempt is made to create a new user account, the same message can be displayed regardless of whether the email address exists in the system or not.  For example: \"A confirmation has been sent to the email provided.\"",
+              "remediation_advice": "Login: When an authentication request fails, ensure that all responses are identical, regardless of whether it was the username or the password that caused the authentication to fail.  The HTTP response needs to be the same as well. There should not be a single byte or bit difference.\n\nReset Password: There can not be a discrepancy in the form responses when using the reset a password functionality.  The same message should be sent, bach to the user, regardless of whether the username or email exists as an account or not.\n\nRegistration: As a best practice to prevent usernames from being enumerated when creating an account, the username should an email address.  When an attempt is made to create a new user account, the same message can be displayed regardless of whether the email address exists in the system or not.  For example: \"A confirmation has been sent to the email provided.\"",
```

Comment:
1. Double whitespaces between sentences
2. "bach" instead of "back"
3. grammar in "the username should an email address"
4. Let's use single quotes `'` instead of escaped double quotes `\"`, I'm not sure if markdown is going to handle slashes properly

Reply:
I didn't add the `\"` part. So do it anyway?

Reply:
yeah, let's replace it